### ISSUE TITLE
508-defect-3 [COGNITON]: COVID Vaccine - Contact preference must be as clear as possible

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#17088]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user with a strong preference for email or telephone contact, I want to feel good answering the question. I need to know my answer is the one I wanted, and that there's not additional information I might have missed.

## Details

We ask a general question about contact preferences, and then ask a follow on question if users click the "Yes" radio button. I had trouble understanding what was being asked and had to read the question several times. I am proposing we could consolidate the initial contact preference question and the sub-questions into one plain language group. Maybe sometime like the following:

**Question:**
How do you want to be contacted?

**Radio button options:**
  1. Email
  2. Phone
  3. I have no preference

## Acceptance Criteria

- [ ] Team has reviewed and considered the change request
- [ ] Team documents their decision in ticket comments

**If team decides to make the change:**
- [ ] Radio group behaves as expected (focus is not lost on keypress or click, etc.)
- [ ] Radio group is well formed and shows no axe violations

## WCAG or Vendor Guidance (optional)

* [Error Prevention (Legal, Financial, Data): Understanding SC](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-reversible.html)

## Screenshots or Trace Logs

![Screen Shot 2020-12-08 at 9 40 12 AM](https://user-images.githubusercontent.com/934879/101507105-cff92f00-393b-11eb-81b3-842bc4e341c4.png)

**Comment by 1Copenut** [comment]
Closing this issue because the elements in question were removed.
